src/blocks/icons/components/InboxBellFilled.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react';
+import { IconWrapper } from '../IconWrapper';
+import { IconProps } from '../Icons.types';
+
+const InboxBellFilled: FC<IconProps> = (allProps) => {
+  const { svgProps: props, ...restProps } = allProps;
+  return (
+    <IconWrapper
+      componentName="InboxBellFilled"
+      icon={
+        <svg
+          width="inherit"
+          height="inherit"
+          viewBox="0 0 32 32"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          {...props}
+        >
+          <path
+            fill-rule="evenodd"
```

Comment:
Can be changed to `fillRule` and `clipRole` to follow React JSX
Same for the IconBell component

Reply:
Let's not change svg and use the same svg which we download from figma:

Reasons:
- To avoid changing and maintaining these tags which will bring no benefit to us but will consume more time.
- To avoid any unnecessary/unknown breakages in browsers other than chrome or other older versions of browsers.